src/Perlang.ConsoleApp/Program.cs

```diff
@@ -201,7 +205,7 @@ public static int MainWithCustomConsole(string[] args, IConsole console)
                             replMode: true,
                             standardOutputHandler: console.Out.WriteLine,
                             disabledWarningsAsErrors: disabledWarningsAsErrorsList
-                        ).RunPrompt();
+                        ).RunPrompt(args);
```

Comment:
(`args` is typically empty in these cases, but it doesn't hurt to provide it to future-proof in case it will be non-empty sometime in the future.)